resource.html

```diff
@@ -0,0 +1,1051 @@
+                                            </h5>
+                                            <div class="cmp-teaser__description">
+                                                <p>by Braden Douglas</p>
+                                            </div>
+                                            <div class="cmp-teaser__action-container button">
+                                                    <a class="cmp-teaser__action-link cmp-button" id="teaser-3a8e3acfdb-cta-641eff4586" data-cmp-data-layer="{&quot;teaser-3a8e3acfdb-cta-641eff4586&quot;:{&quot;@type&quot;:&quot;core-components-examples/components/teaser/cta&quot;,&quot;dc:title&quot;:&quot;Call To Action&quot;,&quot;xdm:linkURL&quot;:&quot;/content/core-components-examples/library/core-content/teaser.html&quot;}}" data-cmp-clickable="" href="/content/core-components-examples/library/core-content/teaser.html"><span>Buy Now</span></a>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="container responsivegrid article-slider aem-GridColumn aem-GridColumn--default--12 layout-grid">
+        <div id="container-b0c5fbd9e2" class="cmp-container">
+            <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+            <!-- ARTICLE MAIN TITLE -->
+            <div class="responsivegrid aem-GridColumn title-btn-wrap aem-GridColumn--default--12 layout-grid">
+                <div id="container-56884b3bb11" class="cmp-container">
+                <!-- ARTICLE TITLE -->
+                    <div class="title aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                        <div id="title-bcb754f8400" class="cmp-title">
+                            <h5 class="cmp-title__text">Leading Through a Crisis Video Series</h5>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <!-- ARTICLE SLIDER WRAP -->
+            <div class=" responsivegrid article-slider-wrap aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                <div id="container-9ce1cf14410" class="cmp-container">
+                <!-- ARTICLE SLIDER ITEM -->
+                <div class="container responsivegrid post-item aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                    <div id="container-172911e9c5" class="cmp-container">
+                    <div class="teaser aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                        <div class="cmp-teaser">
+                        <div class="cmp-teaser__image post-article__image">
```

Comment:
@Prystal254 The image/content order needs correcting and the anchor needs to be in the right place. And `.post-article__image` can't go here. Custom classes can only go on the root of the component.